WordPress/Classes/Utility/BackgroundTasks/WeeklyRoundupBackgroundTask.swift

```diff
@@ -468,6 +537,27 @@ class WeeklyRoundupBackgroundTask: BackgroundTask {
         operationQueue.addOperation(completionOperation)
     }
 
+    // MARK: - Helpers
+
+    private func scheduleDynamicNotification(site: Blog, stats: StatsSummaryData, completion: @escaping (Result<Void, Error>) -> Void) {
+        let dotComID = site.dotComID?.intValue
+        self.notificationScheduler.scheduleDynamicNotification(
```

Comment:
I think we can take a small step further to replace this function's `Blog` argument with site title and dotCom id. IMO, it's generally a good idea to _not_ pass `NSManagedObject` around.

Reply:
Resolved in 89e99ffc8e986db1afebd314e30007ebfed8da8d